modules/custom/az_migration/migrations/migrate_plus.migration.az_node_person.yml

```diff
@@ -0,0 +1,179 @@
+id: az_node_person
+label: AZ Person Content
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: d7_node
+  node_type: uaqs_person
+  constants:
+    slash: '/'
+
+destination:
+  plugin: entity:node
+  bundle: az_person
+  migration_dependencies:
+    requireed:
+      - az_person_categories
+      - az_person_categories_secondary
+
+process:
+  nid: nid
+  vid: vid
```

Comment:
This causes a similar issue as in the case of user migrations - attempting to preserve the node ID overwrites existing nodes on the destination site.